tests/components/luftdaten/test_config_flow.py

```diff
@@ -60,6 +60,21 @@ async def test_communication_error(hass: HomeAssistant) -> None:
     assert result2.get("step_id") == SOURCE_USER
     assert result2.get("errors") == {CONF_SENSOR_ID: "cannot_connect"}
 
+    with patch("luftdaten.Luftdaten.get_data", return_value=True), patch(
+        "luftdaten.Luftdaten.validate_sensor", return_value=True
```

Comment:
Is the patch path correct, or do we need to patch the path in the config flow? Does it matter?

Reply:
I dunno, I want to extract this in a later stage actually, when I bring tests to the sensors.

Reply:
If it works as expected it probably doesn't matter. But sometimes tests can pass even when the patch isn't applied correctly, so it's good to double check.

Reply:
It is what has been originally here already, hence I kept that without messing around with it at this point.

I'm going to replace it with an actual fixture for the client in followups